src/exception/database.error.ts

```diff
@@ -1,9 +1,8 @@
 import { ErrorsMessages } from '@constants/errorMessages';
 import { BaseError } from './base.error';
 import { HttpStatusCode } from '../constants/httpStatusCode';
```

Comment:
could you please use the same @alias from above? something like:
`import {ErrorsMessages, HttpStatusCode} from @contstants`

Reply:
Corrected, thank you!